stages/org.osbuild.ostree.deploy.container

```diff
@@ -109,6 +120,25 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
 
+            aleph_image_digest = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ .Digest }}"')
+            aleph_image_revision = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.revision\" }}"')
+            aleph_image_source = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.source\" }}"')
+            aleph_image_version = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.version\" }}"')
```

Comment:
It'd be more efficient and probably more legible in code to get all the JSON out at once, and then parse it from Python.